deck-of-cards-kata-solutions/src/test/java/bnymellon/codekatas/deckofcards/list/immutable/GoogleGuavaDeckOfCardsAsListTest.java

```diff
@@ -111,6 +125,16 @@ public void dealHands()
         var jdkHands = this.jdkDeck.dealHands(jdkShuffled, 5, 5);
         var ggHands = this.ggDeck.dealHands(ggShuffled, 5, 5);
         Assertions.assertEquals(jdkHands, ggHands);
+        var hands = Lists.newArrayList(
+                Iterables.concat(Collections.singleton(Collections2.transform(
+                        ggHands, o -> Joiner.on(", ").join(Ordering.natural().sortedCopy(o))))));
+        List<String> expectedHands = org.eclipse.collections.impl.factory.Lists.mutable.with(
```

Comment:
Use `List.of` from JDK or `Lists.newArrayList` from Guava.

Reply:
Removed use of Java.util.Collections library